beast/physicsmodel/dust/tests/test_extinction.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 import pytest
 
-from . import extinction
+from beast.physicsmodel.dust import extinction
```

Comment:
What absolute import?  relative is better to use it without installing into the system.

from .. import extinction